README.md

```diff
@@ -20,11 +20,25 @@ Some common use cases may include testing code without invoking:
 - etc.
 
 # Interface
-The `ioc` package provides the function `iocFactory` which returns an instance of the `ioc` object with the following properties:
-- `productionMode();` disables the use of ioc and forces the use of the original dependencies regardless of if they have been overridden or not
-- `dep();` provides the primary functionality of overriding and resetting dependencies through the following functions:
-  * `dep.set(deps: Record<string, unknown>);` overrides the component level dependencies with the provided stubs
-  * `dep.reset();` restores dependencies to their original state
+The package provides two methods to implement `ioc` in your code, the deps method and the defaults method. The deps method is implemented with the `ioc.deps()` function. 
```

Comment:
The word 'methods' is overloaded in this context, since a `method` is, in JS "a function on an object". Maybe `ways of implementing`? Some guidance on which 'workflow' I might prefer might also be handy.